config.c

```diff
@@ -156,25 +156,35 @@ static int config_get_value_bl_type(struct dl_list *config_list, char *key,
 	return value;
 }
 
-static int config_parse_log_server_outputs(char *value)
+static int config_parse_log_server_outputs(const char *value)
 {
 	char *token, *tmp;
 	int server_outputs = 0;
 
-	for (token = strtok_r(value, ",", &tmp); token;
+	char *val = strdup(value);
+
+	for (token = strtok_r(val, ",", &tmp); token;
 	     token = strtok_r(NULL, ",", &tmp)) {
-		if (!strcmp(token, "singlefile"))
+		if (!strncmp(token, "singlefile", strlen("singlefile")))
 			server_outputs |= LOG_SERVER_OUTPUT_SINGLE_FILE;
-		else if (!strcmp(token, "filetree"))
+		else if (!strncmp(token, "filetree", strlen("filetree")))
 			server_outputs |= LOG_SERVER_OUTPUT_FILE_TREE;
-		else if (!strcmp(token, "nullsink"))
+		else if (!strncmp(token, "nullsink", strlen("nullsink")))
 			server_outputs |= LOG_SERVER_OUTPUT_NULL_SINK;
-		else if (!strcmp(token, "stdout"))
+		else if (!strncmp(token, "stdout.containers",
```

Comment:
aren't we mixing in and out here? wouldn't be more appropriate to have a log.capture... like we do with dmesg as container and pantavisor are log sources, while log.server.outputs describe how the output is produced?